Resources/config/security.xml

```diff
@@ -6,7 +6,6 @@
 
     <parameters>
         <parameter key="fos_user.security.interactive_login_listener.class">FOS\UserBundle\EventListener\LastLoginListener</parameter>
-        <parameter key="fos_user.security.login_manager.class">FOS\UserBundle\Security\LoginManager</parameter>
```

Comment:
I'm curious. Why this is BC break?

I tried to follow [this advice](https://github.com/FriendsOfSymfony/FOSOAuthServerBundle/pull/315#discussion_r29235933).

What do you think @stof?